lib/handlers/router-entities/route-caching/cached-routes-configuration.ts

```diff
@@ -0,0 +1,50 @@
+import { TradeType } from '@uniswap/sdk-core'
+import { CacheMode, ChainId } from '@uniswap/smart-order-router'
+import { CachedRoutesStrategy } from './model/cached-routes-strategy'
+import { PairTradeTypeChainId } from './model/pair-trade-type-chain-id'
+import { CachedRoutesParameters } from './model/cached-routes-parameters'
+
+/**
+ * This is the main configuration for the caching strategies of routes.
+ *
+ * The keys of maps is generated out of calling the `toString` method in the `PairTradeTypeChainId` class,
+ * this way we can guarantee the correct format of the key.
+ *
+ * The values are an object of type `CachedRoutesStrategy` which is initialized with an array of `CachedRoutesParamteres`
+ */
+export const CACHED_ROUTES_CONFIGURATION: Map<string, CachedRoutesStrategy> = new Map([
+  [
+    new PairTradeTypeChainId({
+      tokenIn: '0xc02aaa39b223fe8d0a0e5c4f27ead9083c756cc2', // WETH
+      tokenOut: '0xa0b86991c6218b36c1d19d4a2e9eb0ce3606eb48', // USDC
+      tradeType: TradeType.EXACT_INPUT,
+      chainId: ChainId.MAINNET,
+    }).toString(),
+    new CachedRoutesStrategy([
+      new CachedRoutesParameters({ bucket: 1, blocksToLive: 2, cacheMode: CacheMode.Tapcompare }),
+      new CachedRoutesParameters({ bucket: 5, blocksToLive: 2, cacheMode: CacheMode.Tapcompare }),
+      new CachedRoutesParameters({ bucket: 10, blocksToLive: 1, cacheMode: CacheMode.Tapcompare }),
+      new CachedRoutesParameters({ bucket: 50, blocksToLive: 1, cacheMode: CacheMode.Tapcompare }),
+      new CachedRoutesParameters({ bucket: 100, blocksToLive: 1, cacheMode: CacheMode.Tapcompare }),
+      new CachedRoutesParameters({ bucket: 500, blocksToLive: 1, cacheMode: CacheMode.Tapcompare }),
```

Comment:
Coming back to this, these buckets seem far too large right? I.e. a 1.01 WETH -> USDC would hit the 5 WETH -> USDC one?

Reply:
probably yeah, although I don't expect it to be too different. I pulled these number from the boundaries in the TrackedPairsDashboard from routing-api, but yeah we could create different buckets